website/www/site/layouts/shortcodes/list_with_icons.html

```diff
@@ -0,0 +1,29 @@
+{{/*
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License. See accompanying LICENSE file.
+   */}}
+   
+    {{ $data := index $.Site.Data .Site.Language.Lang }}
+    <div class="icon-list">
+        {{ range $item := $data.community_list }}
```

Comment:
Just want to ping this again, there's the same problem with the `table_with_icons` shortcode, where `$data.contact_us` is hard-coded.

Reply:
Yes, you are right, it could be done with more reusability, I pushed fixed version

Reply:
Thanks this looks better! `table_with_icons` is still a little bit hard-coded since it assumes a "purpose", "content" and "action" column. Could you look at making it accept a list of column names as an argument? If that's too tricky, please just put a comment in the file to describe the assumption.

Reply:
Of course, I made appropriate changes